PowerUp/app/src/main/res/layout/gamemap.xml

```diff
@@ -39,21 +39,22 @@
         android:id="@+id/BoyfriendButton"
         android:layout_width="@dimen/map_click_width"
         android:layout_height="@dimen/map_click_height"
-        android:layout_above="@id/ContinueButtonMapGameBegins"
-        android:layout_alignLeft="@id/HouseButton"
-        android:layout_alignStart="@id/HouseButton"
+
         android:alpha="0.00"
-        android:text="@string/boyfriend" />
+        android:text="@string/boyfriend"
+        android:layout_alignTop="@+id/SchoolButton"
+        android:layout_alignLeft="@+id/HouseButton"
+        android:layout_alignStart="@+id/HouseButton" />
```

Comment:
Use `@+id` only when declaring an id using `android:id`. If you are refering to an existing object as you are doing here, use `@id` instead.